keyboards/handwired/mingwahlu/mingwahlumacro/keymaps/default/keymap.c

```diff
@@ -0,0 +1,184 @@
+                SEND_STRING("QMK is the best thing ever!");
+            } else {
+                // when keycode QMKBEST is released
+            }
+            break;
+        case QMKURL:
+            if (record->event.pressed) {
+                // when keycode QMKURL is pressed
+                SEND_STRING("https://qmk.fm/\n");
+            } else {
+                // when keycode QMKURL is released
+            }
+            break;
+        case M_NUTS:
+            if (record->event.pressed) {
+                SEND_STRING("deeznuts");
+                tap_code(KC_ENTER);
+            } else {
+            }
+            break;
+        case M_RESET:
+            if (record->event.pressed) {
+                reset_keyboard();
+            } else {
+            }
+            break;
+        case M_CLOSETAB:
+            if(record->event.pressed) {
+                SEND_STRING(SS_LCTL("w"));
+            } else {
+
+            }
+            break;
+        case M_REOPENTAB:
+            if(record->event.pressed) {
+                SEND_STRING(SS_LCTL(SS_LSFT("t")));
+            } else {
+
+            }
+            break;
+        case M_NEWTAB:
+            if(record->event.pressed) {
+                SEND_STRING(SS_LCTL("t"));
+            } else {
+
+            }
+            break;
+        case M_SWITCHTABS:
+            if(record->event.pressed) {
+                register_code(KC_LCTL);
+                tap_code(KC_TAB);
+            } else {
+               unregister_code(KC_LCTL);
+            }
+            break;
+        case M_ALTTAB:
+            if(record->event.pressed) {
+                register_code(KC_LALT);
+                tap_code(KC_TAB);
+            } else {
+                unregister_code(KC_LALT);
+            }
+            break;
+        case M_ALTF4:
+            if(record->event.pressed) {
+                register_code(KC_LALT);
+                tap_code(KC_F4);
+            } else {
+               unregister_code(KC_LALT);
+            }
+            break;
+        case M_UMARUBM:
+            if(record->event.pressed) {
+                SEND_STRING("https://imgur.com/NvKoahR");
+                tap_code(KC_ENTER);
+            } else {
+            }
+            break;
+        case M_UMARUEAT:
+            if(record->event.pressed) {
+                SEND_STRING("https://tenor.com/view/ramen-slurp-anime-umaru-cute-gif-16722331");
+                tap_code(KC_ENTER);
+            } else {
+            }
+            break;
+        case M_UMARUMAD:
+            if(record->event.pressed) {
+                SEND_STRING("https://tenor.com/view/anime-umaru-mad-cute-small-girl-gif-5169671");
+                tap_code(KC_ENTER);
+            } else {
+            }
+            break;
+        case M_UMARUHAPPY:
+            if(record->event.pressed) {
+                SEND_STRING("https://tenor.com/view/umaru-gif-4927713");
+                tap_code(KC_ENTER);
```

Comment:
```suggestion
                SEND_STRING("https://tenor.com/view/umaru-gif-4927713" SS_TAP(X_ENTER));
```

Reply:
all other macros have been changed to use this layout  
```c
SEND_STRING("url" SS_TAP(X_ENTER));
```